lib/flux/settings-reducer.js

```diff
@@ -49,11 +49,25 @@ const markdownEnabled = ( state = false, action ) => {
 	return action.markdownEnabled;
 };
 
+const sourceList = ( state = { width: 330, isResizing: false, canResize: false }, { type, width, canResize, isResizing } ) => {
```

Comment:
I'm a bit leery of the length of this line already. I _like_ how the action is restructured, but I think, "what if we add one more property?" when do we split it to a new line?

In reducers like these, I often prefer to keep `action` in the props then restructure inside. As far as the initial state goes I have less of an opinion, but I'm going to go ahead and give a counter-example.

```js
const initialSourceList = {
	width: 330,
	isResizing: false,
	canResize: false
}
const sourceList( state = initialSourceList, action ) {
	// now if we get more, we just start putting one item
	// per line - we could do this already too I suppose
	const { type, width, canResize, isResizing } = action;

	// or for more obvious diffs
	const {
		canResize,
		isResizing,
		type,
		width
	} = action;

	…
}
```